test/unit/sprites_rendered-target.js

```diff
@@ -280,6 +280,22 @@ test('isTouchingPoint', t => {
     t.end();
 });
 
```

Comment:
Thanks for adding this test. Could you also please add a test where the sprite _is_ touching the mouse?

Reply:
Not sure how to add that test?

Reply:
```js
test('isTouchingMouse', t => {
    const r = new Runtime();
    const s = new Sprite(null, r);
    const renderer = new FakeRenderer();
    r.attachRenderer(renderer);
    const a = new RenderedTarget(s, r);
    a.renderer = renderer;
    r.ioDevices.mouse.postData({
        x: 0,
        y: 0
    });
    t.equals(a.isTouchingObject('__mouse__'), true);
    t.end();
});
```

I think this will work because `FakeRenderer.drawableTouching` always returns `true`